FashionConnect_api/routes/users.js

```diff
@@ -80,17 +79,17 @@ router.post('/users/profile', async (req, res) => {
       const user = await User.findOne({ where: { username:username } });
       console.log(user)
   
-      if (!user) {
-        return res.status(404).json({ error: 'User not found' });
-      }
-
-      user.First_Name = First_Name
+      user.First_Name =First_Name
       user.Last_Name = Last_Name
       await user.save()
   
-      // Return the user data in the response
-      res.json({ message: "User profile uodated succesfully" });
-    } catch (error) {
+      res.json({ message: "User info updated" });
+
+        return res.status(404).json({ error: 'User not found' });
```

Comment:
Is this correct? Looks like the update was successful but you are returning an error response?